docs/site/Debugging-with-vscode.md

```diff
@@ -0,0 +1,18 @@
+---
+lang: en
+title: 'Debugging with VS Code'
+keywords: LoopBack 4.0, LoopBack 4, Node.js, TypeScript, Debug
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Debugging-with-vscode.html
+---
+
+If you are using Visual Studio Code, below are some tips to help you debug your
+LoopBack applications.
+
+The monorepo `loopback-next` contains a folder `.vscode` at root level for
+debugging the test files. When start debugging, it will stop at any breakpoint
+triggered by the tests. There are two predefined configurations defined in file
+`.vscode/launch.json`:
+
+- Run Mocha tests: Debug all tests inside the `/packages` folder.
```

Comment:
I think the original intention was that "Run Mocha tests" should run tests in all monorepo packages, including examples etc. Because otherwise how would you run the entire test suite? 

Do you mind to fix the task config to include the following directories?

```
fixtures/*,
benchmark,
examples/*,
packages/*,
extensions/*,
sandbox/*
```

I think there are not tests inside `docs` and `acceptance` tests require a special setup (e.g. start MongoDB in a Docker container), so these directories should be excluded.

Reply:
@bajtos Hmm can we discuss and do it in a separate story? Actually I tend to run fewer tests per task and if people really want to run others, we can write instructions of modifying the task entry, or create new tasks.